TACC/Core-Portal
# Reconfigure local dev to use cep.test

## Overview
Use `cep.test` instead of `cep.dev` locally to avoid collision with the actual `cep.dev` TLD.

## Testing

1. Make sure that your `/etc/hosts` file includes `127.0.0.1... cep.test`
2. Copy the secret settings from the "[Core Portal Settings Secret](https://stache.utexas.edu/entry/bedc97190d3a907cb44488785440595c)" Stache entry and uncomment the `cep.test client config` block.
3. Start your local dev environment as usual and access the portal at `https://cep.test`.
